controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -1000,12 +1003,13 @@ func (a *ExistingInfraMachineReconciler) modifyNode(ctx context.Context, nodeNam
 		updateErr := a.Client.Update(ctx, &result)
 		if updateErr != nil {
 			contextLog.Errorf("failed attempt to update node: %v", updateErr)
+			time.Sleep(2 * time.Second)
```

Comment:
Please add a comment here.  When someone encounters this later they will be very tempted to remove it.